tests/test_PLKSR.py

```diff
@@ -0,0 +1,119 @@
+from spandrel.architectures.PLKSR import PLKSR, PLKSRArch, RealPLKSR
+
+from .util import (
+    ModelFile,
+    TestImage,
+    assert_image_inference,
+    assert_loads_correctly,
+    assert_size_requirements,
+    disallowed_props,
+    skip_if_unchanged,
+)
+
+skip_if_unchanged(__file__)
+
+
+def test_load():
+    assert_loads_correctly(
+        PLKSRArch(),
+        lambda: PLKSR(),
+        lambda: PLKSR(dim=32),
+        lambda: PLKSR(upscaling_factor=2),
+        lambda: PLKSR(n_blocks=6),
+        lambda: PLKSR(kernel_size=9),
```

Comment:
Those aren't enough tests. You have a lot of logic for detecting e.g. `ccm_type` and none if it is tested here. Every parameter should be tested, and if a parameter has only a few possible values, all possible values should be tested.